crates/orderbook/src/orderbook.rs

```diff
@@ -243,6 +244,7 @@ impl Orderbook {
         domain_separator: DomainSeparator,
         settlement_contract: H160,
         database: crate::database::Postgres,
+        database_replica: crate::database::Postgres,
```

Comment:
We should be more specific here probably to avoid any confusion in the future.
```suggestion
        db_write: crate::database::Postgres,
        db_read: crate::database::Postgres,
```
The naming is not great, tho, feel free to use your option.